openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -140,7 +145,6 @@ def get_cohorted_commentables(course_key):
     return ans
 
 
-@transaction.commit_on_success
```

Comment:
All the transaction handling is now done inside `membership.save()`, it' no longer needed here.